### ISSUE TITLE
[rule] `link-min-length-4` not properly identifying content?

### ISSUE THREAD

**Issue by acalvo** [linthtml/linthtml#142]
Not sure if it's a bug, but I found this other issue trying LintHTML against the HTML of our projects:

```html
<a href="https://google.com"><span>Google</span></a>
```

This code snippet triggers an error in the `link-min-length` rule since the content is wrapped in a `span` tag. Is it expected behavior?

**Comment by KamiKillertO** [comment]
Thanks for the report it's fixed now :+1:
